utils/default.go

```diff
@@ -0,0 +1,94 @@
+package utils
+
+import (
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+func tagHandlers(field reflect.Value, tagValue string) {
+	//nolint:exhaustive // We don't need to handle all types
+	switch field.Kind() {
+	case reflect.String:
+		if field.String() == "" {
+			field.SetString(tagValue)
+		}
+	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
+		if field.Int() == 0 {
+			if i, err := strconv.ParseInt(tagValue, 10, 64); err == nil {
+				field.SetInt(i)
+			}
+		}
+	case reflect.Float32, reflect.Float64:
+		if field.Float() == 0.0 {
+			if f, err := strconv.ParseFloat(tagValue, 64); err == nil {
+				field.SetFloat(f)
+			}
+		}
+	case reflect.Bool:
+		if !field.Bool() {
+			if b, err := strconv.ParseBool(tagValue); err == nil {
+				field.SetBool(b)
+			}
+		}
+	case reflect.Slice:
```

Comment:
Can you add support for single and multiple pointers here?

Reply:
Okey, I will add it newt commit.